packages/components/src/components/Overall/DataSummary.module.scss

```diff
@@ -67,7 +68,36 @@
 
 .container {
   display: flex;
-  flex-direction: column;
-  justify-content: center;
-  margin-bottom: 16px;
+  align-items: center;
+  padding: 10px 20px 10px 5px;
+  border-radius: 10px;
+  margin: 10px 0;
+
+  .icon {
+    margin: 0 5px 0 10px;
+  }
+
+  .description {
+    margin-right: 5px;
+    font-family: PingFang SC;
```

Comment:
Usually we use multiple fonts in font-family because different operating systems have different built-in fonts.

Reply:
I recommend using the global font-family for most elements. Only override font-family for code block or special numbers.